### PR TITLE
Fix signature to re-enable transition cost for motorcycle

### PR DESCRIPTION
**Pull request by SvetlanaBayarovich**
# Issue

Method signature got inconsistent with the one of the base class. As a result, zero transition cost was applied for motorcycle profile. 

## Tasklist

 - [ ] Add tests
 - [x] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [x] Update the [changelog](CHANGELOG.md)

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

**Review by danpat — DISMISSED** [review]

**Review by gknisely — DISMISSED** [review]

File: src/sif/motorcyclecost.cc [hunk 1]
```diff
@@ -242,8 +241,7 @@ class MotorcycleCost : public DynamicCost {
   virtual Cost TransitionCostReverse(const uint32_t idx,
                                      const baldr::NodeInfo* node,
                                      const baldr::DirectedEdge* pred,
-                                     const baldr::DirectedEdge* edge,
-                                     const bool has_traffic) const;
+                                     const baldr::DirectedEdge* edge) const override;
```
Comment (mandeepsandhu) [inline src/sif/motorcyclecost.cc:244]: You can remove comment for `has_traffic` from both function's doc comments.

**Review by mandeepsandhu — DISMISSED** [review]
1 minor comment. Rest looks good :+1:

File: src/sif/motorcyclecost.cc (hunk 1 above)
Comment (SvetlanaBayarovich) [reply src/sif/motorcyclecost.cc:244]: Good catch! 👍

**Review by gknisely — APPROVED** [review]

**Review by mandeepsandhu — APPROVED** [review]

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,7 @@
    * FIXED: Mark the restriction start and end as via ways to fix IsBridgingEdge function in Bidirectional Astar [#2796](https://github.com/valhalla/valhalla/pull/2796)
    * FIXED: Dont add predictive traffic to the tile if it's empty [#2826](https://github.com/valhalla/valhalla/pull/2826)
    * FIXED: Fix logic bidirectional astar to avoid double u-turns and extra detours [#2802](https://github.com/valhalla/valhalla/pull/2802)
+   * FIXED: Re-enable transition cost for motorcycle profile [#2837](https://github.com/valhalla/valhalla/pull/2837)
```
Comment (kevinkreiser) [inline CHANGELOG.md:13]: `s/profile/costing`

**Pull request merged**